templates/workspace_services/guacamole/template_schema.json

```diff
@@ -66,7 +66,7 @@
             "arrayMatchField": "name",
             "value": {
               "name": "{{ resource.id }}",
-              "value": "{{ resource.properties.connection_uri }}"
```

Comment:
why removing connection uri?

Reply:
Since connection_uri isn't what is needed when setting the aad app redirect urls.